src/views/ComponentGraphView.vue

```diff
@@ -0,0 +1,111 @@
+<script setup lang="ts">
```

Comment:
Can give some info what this test component used for (because there's a static data)

Reply:
This in only for test using static data, I'm thinking that ideally every component can be viewed standalone like this, and when we add them to other component, the component should look the same.

Reply:
this will be deleted later once we complete developing

Reply:
maybe add some information about this component (example for graph view) not that bad?

Reply:
ok